doc/user/content/overview/_index.md

```diff
@@ -10,185 +10,66 @@ menu:
     weight: 5
 ---
 
-Materialize is a **streaming database** powered by [Timely](https://github.com/TimelyDataflow/timely-dataflow#timely-dataflow) and
+Materialize is a **streaming database** purpose-built for low-latency
+applications. Data and software engineering teams use it to solve problems
+where data must be processed at speeds and scales not possible in traditional
+databases, but without the cost, complexity, and development time of most
+streaming engines.
+
+If you're looking to reduce the time it takes to refresh answers to common
+queries, Materialize can make that happen! Rather than recalculating results
+from scratch each time it's queried, or serving stale cached results,
+Materialize continually ingests data and keeps results up to date for you
+as **incrementally updated materialized views**.
+
+## Features
+
+Materialize combines all the things you love about databases and abstracts all
+the hard parts that go into making a streaming engine horizontally scalable,
+highly available and strongly consistent. To get up and running, all you need
+to know is **SQL**.
+
+### Incrementally updated views
+
+Materialized views are useful to avoid re-running heavy queries that are asked
```

Comment:
Most users will be familiar with the concept of materialized views, so this section can still be compacted a bit more.